includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -191,6 +196,39 @@ private function get_module_block_html( $block ) {
 		';
 	}
 
+	/**
+	 * Get progress indicator HTML.
+	 *
+	 * @param array $module_id The module id.
+	 * @param int   $course_id The course id.
+	 *
+	 * @return string Module HTML
+	 */
+	private function get_progress_indicator( $module_id, $course_id ) {
+
+		$module_progress = Sensei()->modules->get_user_module_progress( $module_id, $course_id, get_current_user_id() );
+
+		if ( empty( $module_progress ) ) {
+			return '';
+		}
+
+		if ( $module_progress < 100 ) {
+			$module_status   = __( 'COMPLETED', 'sensei-lms' );
```

Comment:
I wonder if it'd be better to put these like `Completed`/`In Progress` and then use `text-transform: uppercase;` in CSS to get the effect.

Reply:
Good idea! a5be66aed915883560496029e5e30ec02361a976